includes/payment-methods/class-upe-payment-gateway.php

```diff
@@ -599,7 +605,8 @@ public function payment_fields() {
 				<input id="wcpay-payment-method-upe" type="hidden" name="wcpay-payment-method-upe" />
 
 			<?php
-			if ( $this->is_saved_cards_enabled() ) {
+			$saveable_payment_methods = array_filter( $this->get_upe_enabled_payment_method_ids(), [ $this, 'is_enabled_for_saved_payments' ] );
```

Comment:
I'm not sure if we have this option within the current settings (at least I am not checking for this scenario at this point), but I'm pretty sure it was there in the old settings, so maybe it might come back. I've no issue with updating this variable name, in fact I've transformed my emotional commitment into a GitHub commit, and you should find this update in the latest changes.